docs(bil): Added some concepts MTA-4068 (#2652)
scaleway/docs-content: console/billing/concepts.mdx

content:
  h1: Billing - Concepts
  paragraph: This page explains all the concepts related to Billing
tags: billing account pricing payment
categories:
  - account
  - console
---

> 
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Billing alerts 

The Billing Alerts functionality within the Scaleway console allows users to oversee and control their expenses efficiently. This feature allows you to create alerts that notify users when predefined budget thresholds are met. Your alerts can be seamlessly delivered through SMS, email, or API webhook channels.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Billing API

Scaleway's Billing API provides a quick overview of your cloud consumption and allows you to monitor expenses programmatically.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Billing contact

The billing contact is an additional contact who will receive the invoice or payment alerts by email. This allows you to add a person who may not have access to the Scaleway console, but who should be aware of billing, payment documents and notifications.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Billing cycle

Invoices are issued each month based on the resource consumption from the previous month. Our reference billing period is the definition of a calendar month in the Coordinated Universal Time (UTC) time zone. The automatic payment of invoices occurs in the first few days of the month, once the invoice is issued and processed using the preferred payment method of your account.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Billing information

Consists of information about the organization that will be billed. They will appear on the feature and can be modified on the Scaleway console.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Consumption pie chart

> The consumption pie chart reflects the amount of money spent on the products you've used. Its monetary value is calculated in real-time, providing an instant overview of your expenses.

Reviewer 2: maybe include an image

> The consumption pie chart reflects the amount of money spent on the products you've used. Its monetary value is calculated in real-time, providing an instant overview of your expenses.

Author: > maybe include an image

I am going to leave the screenshot for later.. will wait for the launch of the billing space

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Cost Manager 

> Cost manager is a comprehensive and user-friendly tool to monitor and analyze cloud resource consumption, costs, and trends. It will offer a clear and detailed view of consumption data, enabling users to accurately reconcile costs for re-invoicing purposes.

Reviewer 2: if it is not out yet, it's best that we don't talk about it yet. unless it is planned to be out by the time the doc is published. if this is the case then you should write it in the present tense 🙏🏻

> Cost manager is a comprehensive and user-friendly tool to monitor and analyze cloud resource consumption, costs, and trends. It will offer a clear and detailed view of consumption data, enabling users to accurately reconcile costs for re-invoicing purposes.

Author: > Future tense - is this a future feature or something that is already available?

I have written the concept in the present tense. The feature is not available yet but I will still keep in the doc for the moment.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Free Tier 

> The Free Tier offers certain products at no cost and up to a specified usage limit. These limits do not have an expiration date but may be subject to change. Eligible customers can enjoy these select services free of charge when creating an account, with a usage limit in place.

Reviewer 3: I don't think free tier should be capitalized but not sure - to check with @jsudraud

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Free Trial

> The Free Trial provides users access to specific products at no cost. However, it is available only for a limited duration and usage quantity.

Reviewer 3: ```suggestion
A free trial provides users access to specific products at no cost. However, it is available only for a limited duration and usage quantity.
```

Reviewer 3: In what circumstances do we offer a free trial?

Reviewer 2: I think this refers to Object Storage only, for now. you can tell them it's available for OS and include a link to the doc page that talks about the free trial

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## IAM billing permission

> Each group is linked to a permission set. There are two permission sets linked to billing:

Reviewer 3: ```suggestion
IAM billing permissions can be given to users to allow them to read and/or manage the Organization's billing. There are two permission sets linked to billing:
```

Billing read only: List and read access to billing information. This allows the user to view consumption, invoices, discounts, budget alerts and payment methods.
Billing Manager: Full access to billing management. This allows the user to list, read and edit billing contact information, payment information, billing alerts and invoices.

> 

Reviewer 3: You can give users permissions via [policies](/identity-and-access-management/iam/how-to/create-policy/), or by adding them to a [group](/identity-and-access-management/iam/how-to/manage-users/#how-to-manage-a-user's-groups) already attached to a relevant policy.

Reviewer 2: you can add what Rowena wrote in a note box

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Pay as you go 

> Public cloud services follow a 'pay as you go' model, ensuring you pay only for what you use. Some products offer plan options. Your monthly invoice, calculated at the end of the month, is based on exact resource usage and may include additional invoices for upfront payments during the month.

Reviewer 3: I think it might be good to point out that Scaleway is one of these public cloud services?

> Public cloud services follow a 'pay as you go' model, ensuring you pay only for what you use. Some products offer plan options. Your monthly invoice, calculated at the end of the month, is based on exact resource usage and may include additional invoices for upfront payments during the month.

Reviewer 3: ```suggestion
Public cloud services follow a 'pay as you go' model, ensuring you pay only for what you use. Some products offer additional plan options. Your monthly invoice, calculated at the end of the month, is based on exact resource usage and may include additional invoices for upfront payments during the month.
```

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Payment methods
 
The payment methods registered on your account are used to pay your invoice. You can set one of your payment methods as the default and manage your different payment methods from the console.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Project

A Project is a grouping of Scaleway [resources](#resource). Each Scaleway Organization comes with a default Project, and you can create new Projects if necessary. Projects are cross-regional, meaning resources located in different [regions](/compute/instances/concepts/#region) can be grouped into one single Project. When grouping resources into different Projects, you can use [IAM](/identity-and-access-management/iam/concepts/#iam) to define custom access rights for each Project.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Pro-rata billing

You are billed pro-rata on Dedibox and Web Hosting services for the initial month of a subscription. This means the charge is adjusted to reflect the portion of the first month you used the service. Starting from the second month and onwards, you are billed in full at the beginning of each month, covering the entire upcoming billing period in advance. This billing model ensures that you pay for the service based on your actual usage for the first month and then in advance for subsequent months.

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

## Support plan 

Scaleway provides four different types of [support plans](https://console.scaleway.com/support/plans): Basic, Bronze, Silver and Gold. Your support plan determines the level of service and dedicated assistance you have access to, and the guaranteed response time of your support requests. You can [configure your support plan in the console](/console/account/how-to/configure-support-plans/).

> 
> </Concept>
> 
> <Concept opened>

Reviewer: ```suggestion
```

<Concept opened>

## Voucher

> </Concept>

Author: to be removed

Reviewer 4: It's already removed (that's why the line is red) :)

> A Scaleway voucher provides you with a discount or credits to use on our services. You receive it as part of a promotional campaign or from your account manager. To redeem a voucher, you need to [create a Scaleway account](/console/account/how-to/create-an-account) and have a valid payment method. Voucher codes are unique and valid only for a defined period.

Reviewer 3: ```suggestion
A Scaleway voucher provides you with a discount or credits to use on our services. You may receive one as part of a promotional campaign or from your account manager. To redeem a voucher, you need to [create a Scaleway account](/console/account/how-to/create-an-account) and have a valid payment method. Voucher codes are unique and valid only for a defined period.
```

> </Concept>

Reviewer: ```suggestion
```

</Message>

## Pay as you go 

Scaleway offers public cloud services that follow a 'pay as you go' model, ensuring you pay only for what you use. Some products offer additional plan options. Your monthly invoice, calculated at the end of the month, is based on exact resource usage and may include additional invoices for upfront payments during the month.

## Payment methods
 
The payment methods registered on your account are used to pay your invoice. You can set one of your payment methods as the default and manage your different payment methods from the console.

## Project

A Project is a grouping of Scaleway [resources](/en/docs/identity-and-access-management/iam/concepts/#resource). Each Scaleway Organization comes with a default Project, and you can create new Projects if necessary. Projects are cross-regional, meaning resources located in different [regions](/compute/instances/concepts/#region) can be grouped into one single Project. When grouping resources into different Projects, you can use [IAM](/identity-and-access-management/iam/concepts/#iam) to define custom access rights for each Project.

## Pro-rata billing

> You are billed pro-rata on Dedibox and Web Hosting services for the initial month of a subscription. This means the charge is adjusted to reflect the portion of the first month you used the service. Starting from the second month and onwards, you are billed in full at the beginning of each month, covering the entire upcoming billing period in advance. This billing model ensures that you pay for the service based on your actual usage for the first month and then in advance for subsequent months.

Reviewer 5: Il va y avoir des evolution reguliere sur les produits avec prorata, peut ton ne pas donner de nom de produit ici ?

## Support plan 

> Scaleway provides four different types of [support plans](https://console.scaleway.com/support/plans): Basic, Bronze, Silver and Gold. Your support plan determines the level of service and dedicated assistance you have access to, and the guaranteed response time of your support requests. You can [configure your support plan in the console](/console/account/how-to/configure-support-plans/).

Reviewer 5: Support plan are actually: Basic, Silver, gold, Platinum
